adapters/e_volution/evolution.go

```diff
@@ -0,0 +1,112 @@
+package evolution
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type adapter struct {
+	URI string
+}
+
+type bidExt struct {
+	MediaType openrtb_ext.BidType `json:"mediaType,omitempty"`
```

Comment:
Nitpick: I missed this earlier. The `omitempty` part of the tag is only relevant for marshaling. Since you are only unmarshaling with this struct, you can get away with the slightly simpler:

```
type bidExt struct {
	MediaType string `json:"mediaType"`
}
```